samples/vboxmonitor/Makefile

```diff
@@ -2,14 +2,31 @@
 
 # Change the following to match your installation
 BOINC_DIR = ../..
-BOINC_API_DIR = $(BOINC_DIR)/api
-BOINC_LIB_DIR = $(BOINC_DIR)/lib
-
-CXXFLAGS = -g \
+BOINC_SOURCE_API_DIR = $(BOINC_DIR)/api
```

Comment:
You probably added it here for consistency but this is not supposed to be build for Android.
I'd probably removed redundant code for Android

Reply:
I keep